IgOmeProfiling_pipeline.py

```diff
@@ -180,7 +182,16 @@ def run_pipeline(fastq_path, barcode2samplename_path, samplename2biologicalcondi
                              '--no_rpm_factor' if no_rpm_factor else '',
                              '--no_use_rpm_faa_scanning' if no_use_rpm_faa_scanning else '',
                              f'--no_output_sequences_scanning' if no_output_sequences_scanning else '',
-                             f'--error_path {error_path}', '-v' if verbose else '', f'-q {queue}','-m' if use_mapitope else '']        
+                             '--use_positive_motifs_script' if use_positive_motifs_script else '',
+                             '' if invalid_mix is None else f'--invalid_mix {invalid_mix}',
+                             '' if threshold_mean is None else f'--threshold_mean {threshold_mean}',
+                             '' if threshold_std is None else f'--threshold_std {threshold_std}',
+                             '' if threshold_median is None else f'--threshold_median {threshold_median}',
+                             f'--min_max_difference' if min_max_difference else '',
+                             f'--normalize_factor {normalize_factor}', f'--normalize_method_hits {normalize_method_hits}',
+                             f'--normalize_section {normalize_section}',
+                             '' if fixed_min is None else f'--fixed_min {fixed_min}',
+                             '' if fixed_max is None else f'--fixed_max {fixed_max}']       

```

Comment:
Argument names are not clear enugh, it's not clear they are part of positive motifs filter.
Also, change `use_positive_motifs_script` to `filter_positive_motifs`